### ISSUE TITLE
Current Session Widget

### ISSUE THREAD

**Issue by jlegind** [NHMDenmark/Mass-Digitizer#16]
## Issue
This was redesigned and is now out of scope.

Current session widget with the following fields being functional: 

- [x] User name "Logged in as"
- [x] ~Date/Time~
- [x] Current Institution 
- [x] Current collection
- [x] Current workstation
- [x] ~"Settings" button~
- [x] "Log out" button

In this general layout and added to the specimen entry form on the right hand side: 
![image](https://user-images.githubusercontent.com/10909008/183816556-8005e9d5-0f78-4e32-866e-33eb3638bfcb.png)

Dependent on #13

### Solution
Four lines are now employed:
- Logged in as (user)
- Institution name
- collection name
- version number
